.github/workflows/release.yml

```diff
@@ -41,30 +50,27 @@ jobs:
               echo "Release should build only from main branch"
               exit 1
           fi
-          if [ -z "$TRAVIS_API_TOKEN" ]; then
-              echo "TRAVIS_API_TOKEN is empty"
-              exit 1
-          fi
-
-          REQUEST_BUILD_BODY="
-          {
-            \"request\": {
-              \"branch\": \"$branch_name\",
-              \"sha\": \"$commit_sha\",
-              \"merge_mode\": \"deep_merge_append\",
-              \"config\": {
-                \"env\": {
-                  \"global\": [
-                    \"RELEASE_DATREE_PROD=true\",
-                    \"RELEASE_VERSION=${{ needs.define_version_job.outputs.version }}\"
-                  ]
-                }
-              }
-            }
-          }
-          "
-
-          REQUEST_BUILD_BODY=$REQUEST_BUILD_BODY ./.github/workflows/request-travis-build-and-wait-until-finished.sh
+      - name: Setup Go environment
+        uses: actions/setup-go@v2.1.4
+        with:
+          go-version: 1.18
```

Comment:
this will get upgraded soon, just make sure that at the time of pushing this pr, you are up to date :)
https://github.com/datreeio/datree/issues/786